webextensions/manifest/optional_permissions.json

```diff
@@ -355,12 +375,32 @@
             }
           }
         },
+        "dns": {
+          "__compat": {
+            "description": "<code>dms</code>",
+            "support": {
+              "chrome": {
+                "version_added": true
```

Comment:
Chrome does not support the dns API on release:

> Note: This API is only available in Chrome Dev. There are no foreseeable plans to move this API from the dev channel into Chrome stable.

https://developer.chrome.com/docs/extensions/reference/api/dns